src/SFA.DAS.CaptureTheFlag.Web/Views/Game/FindGame.cshtml

```diff
@@ -3,14 +3,6 @@
     ViewData["Title"] = "Index";
 }
 
-<h3>
-    Welcome: 
-    <div id="write">
-        <script>var nickName = document.getElementById("write").innerHTML = window.location.search.slice(10)</script>
-    </div>
-    @*<script>document.getElementById("write").innerHTML = window.location.search.replace("?Nickname=", "Welcome: ")</script>*@
-</h3>
-
 <h1>Finding game</h1>
```

Comment:
What are we replacing this with?

Reply:
The nickname was like a guest login, can now be the username which is present on all pages using the partial view _LoginPartial.  The nickname page was requesting input and then allowing access.  Now once the user is logged in they can access the game.